src/asm2wasm.h

```diff
@@ -565,6 +649,16 @@ class Asm2WasmBuilder {
   }
 
   Function* processFunction(Ref ast);
+
+public:
+  CallImport* isDebugInfo(Expression* curr) {
```

Comment:
`is___` makes me thing "boolean". Better name would be just `debugInfo`, which can still be used in a boolean context.